README.md

```diff
@@ -2,38 +2,49 @@
 
 A Sourcegraph extension that provides code intelligence for TypeScript.
 
-## How to deploy the server
+## How to deploy the language server
 
 The extension is configured to talk to a language server deployed somewhere over WebSockets.
 The server is available as a Docker image `sourcegraph/lang-typescript` from Docker Hub.
 
 ### Using Docker
 
-You can run the server locally with:
+1. Run the language server listening on `ws://localhost:8080`:
 
-```sh
-docker run -p 8080:8080 sourcegraph/lang-typescript
-```
+  ```sh
+  docker run -p 8080:8080 sourcegraph/lang-typescript
+  ```
 
-This will make the server listen on `ws://localhost:8080`, which is what you need to set `typescript.serverUrl` to in Sourcegraph settings so the extension connects to it:
+1. In your Sourcegraph settings, set `typescript.serverUrl` to tell the extension where to connect to the language server:
 
-```json
-  "typescript.serverUrl": "ws://localhost:8080"
-```
+  ```json
+    "typescript.serverUrl": "ws://localhost:8080"
+  ```
 
-To allow the server to access a Sourcegraph instance that is also running in Docker, configure `sourcegraph.url` like this in your Sourcegraph settings:
+1. Also set `typescript.sourcegraphUrl` to tell the language server where to connect to Sourcegraph:
```

Comment:
Note this setting is not needed if the server can contact the server under the same URL as the client (before this was mentioned).